pkg/cspc/algorithm/build_csp.go

```diff
@@ -78,6 +78,13 @@ func (ac *Config) GetCSPISpec() (*cstor.CStorPoolInstance, error) {
 		WithFinalizer(types.CSPCFinalizer).
 		WithNewVersion(version.GetVersion()).
 		WithDependentsUpgraded()
+	// check for OpenEBSDisableDependantsReconcileKey annotation which implies
+	// the CSPI should have OpenEBSDisableReconcileLabelKey enabled
+	if ac.CSPC.GetAnnotations()[types.OpenEBSDisableDependantsReconcileKey] == "true" {
+		cspiObj.WithAnnotationsNew(map[string]string{
```

Comment:
Instead of `WithAnnotationsNew` use `WithAnnotations`. So in the future, if we add `WithannotationNew()` in the above builder we no need to change this line.

Reply:
Done